<!--StartFragment-->

## <p align=center> Key Biomarkers for HighGrade Serous Ovarian Carcinoma </p>

### <p align=center>Literature review of  “The Cancer Genome Atlas Research Network. Integrated genomic analyses of ovarian carcinoma.”  _Nature_ 474, 609–615 (2011). [**https://doi.org/10.1038/nature10166**](https://doi.org/10.1038/nature10166) </p>

Authors (@slack): Vidhyavathy Nagarajan(@Vidhya2205), Hayford Osei Offei(@Hayford72), Emmanuel Afolayemi(@EA(NOVACHRONO), Samuel Aladegbaiye (@Samuel…) 

Github Link: https\://github.com/vidhya2205/hackbio-cancer-internship/blob/stage-1/Stage\_1%20-%20Literature%20Review\.md

LinkedIn Post Link: https://www.linkedin.com/posts/vn2_hackbio-biomarkeridentification-ovariancarcinoma-activity-7238448178558070784-wmPI?utm_source=share&utm_medium=member_desktop

### 1. Introduction- Key objective of the study

High-grade serous ovarian cancer (HGS-OvCa) accounts for about 70% of ovarian cancer deaths, with a low survival probability of 31% post-therapy. This study aims to provide a comprehensive analysis of clinically annotated HGS-OvCa to identify genomic and epigenomic abnormalities, address gaps in understanding their pathophysiology, and explore therapeutic strategies for improved outcomes.

### 2. Methodology - A detailed workflow of how the analysis was performed is presented in Fig1.  below.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXeJrEli2uKheRdhl3pu76pUi9csAdo1GF2pWYyG6Nb91MOUfHNULtf0A_GodNy-F7OuTI3sJzaavEpBhmhuJq0khA-ZPTerRoeBVDU2U7s6d0XhVTmIlHVEjXDzQol_WXBA-UgAWjfKVU4YlUMr-4Wcy2nG?key=Ys7ebyT0dO9LqzbHP9JeUg)

<p align=center>Fig1. Methodology followed in the study </p>

### 3. Results and discussion - What are the key takeaways from each analysis?

#### 3.1. Mutation Analysis

The mutation analysis identified nine genes with mutations in HGS-OvCa samples, consistent with previous results (Ahmed _et al._, _2010_). The most prevalent mutations were in _TP53_ (96%) and _BRCA1/2_ (22%). The identified mutations significantly impact patient diagnosis and prognosis; for instance, the presence of _TP53_ can be used to assess cancer progression, while _BRCA1/2_ mutations have been associated with increased survival due to better responses to treatments, improving patient outcomes.

#### 3.2 Copy number alterations analysis

The SCNA analysis revealed the presence of previously identified (Etemadmoghadam D  _et al.,_ 2009) amplifications and losses, providing diagnostic markers. Some focal amplifications (22, including _MECOM, MAPK1, CCNE1,_ and _KRAS_, present in \~10% of the cases studied) have identified therapeutic inhibitors. Despite previous associations of _CCNE1_ amplifications with lower survival  (Nakayama N _et al_., 2010), this study attributes it to _BRCA_ W/T status since they were coexisting.

#### 3.3. mRNA and miRNA expression and DNA methylation analysis

Analysis of the expression measurements for 11,864 genes detected \~1,500 naturally variable genes, which yielded four robust expression subtypes, grouped based on the gene content in their clusters. Increased DNA methylation was identified in 168 genes in HGS-OvCa samples compared with their fallopian tube controls. The transcriptional signature analysis of 193 genes from 215 samples revealed 108 and 85 genes that were associated with poor and good survival, respectively. Unlike the mutated _BRCA1_ gene, which revealed a better survival rate than W/T, _BRCA1_ promoter hypermethylation showed similar survival rates to W/T, suggesting that patient survival depends on how the _BRCA1_ is inactivated. Consensus clustering further identified 3 miRNA subtypes, with subtype 1 demonstrating a correlation with improved survival rates.

#### 3.4  Pathway analysis

Gene expression analysis in HGS-OvCa identified deregulated pathways including the FOXM1 transcription factor network (87% of cases), the RB1 pathway (67% in cases), the PI3K/RAS pathway (45% of cases), the NOTCH signaling pathway (22% of cases), and the Homologous Recombination (HR) pathway (51% of cases), which can serve as therapeutic targets. Although _FOXM1_ didn’t contain any genetic/epigenetic alteration, __it and its proliferation-related target genes were found to be consistently overexpressed, which correlates with a high mutation rate of _TP53_ in HGS-OvCa directly responsible for regulating _FOXM1,_ indicating the significance of integrated analysis. 

### 4. Conclusion and future scope

The outcome of the study provides researchers and clinicians with diagnostic, therapeutic, and prognostic biomarkers that help to better stratify (subtypes) HGS-OvCa, design drugs to inhibit new targets, and repurpose existing drugs, eventually improving patient outcomes.

### 5. REFERENCES

1.The Cancer Genome Atlas Research Network. Integrated genomic analyses of ovarian carcinoma. _Nature_ 474, 609–615 (2011). <https://doi.org/10.1038/nature10166>

2\. Nakayama N, Nakayama K, Shamima Y, et al. “Gene amplification CCNE1 is related to poor survival and potential therapeutic target in ovarian cancer.” _Cancer_. 2010;116(11):2621-2634. doi:10.1002/cncr.24987

 3. Etemadmoghadam D, deFazio A, Beroukhim R, et al. Integrated genome-wide DNA copy number and expression analysis identifies distinct mechanisms of primary chemoresistance in ovarian carcinomas. _Clin Cancer Res_. 2009;15(4):1417-1427. doi:10.1158/1078-0432.CCR-08-1564

4\. Ahmed, Ahmed Ashour et al. “Driver mutations in TP53 are ubiquitous in high grade serous carcinoma of the ovary.” _The Journal of pathology_ vol. 221,1 (2010): 49-56. doi:10.1002/path.2696

<!--EndFragment-->
